test_pull_requests

```diff
@@ -1169,13 +1353,19 @@ popd
                 set_mergeable(id, repo, login, comments)
               else
                 if set_not_mergeable(id, repo, login) == 0
+                  # TODO: since our counter saturates at 0, why would we ever
+                  # get a counter value of 0 after an increment?
                   mergeability_in_flux = true
                 end
                 next
               end
 
               comments = get_comments(id, repo) if comments.nil?
 
+              # we only want to consider pull requests where the last trigger we found
+              # is from a trusted user
+              #
+              # TODO: use trusted_trigger_time?
```

Comment:
We have to get further in the logic to update the PR with the appropriate messages if the pull request isn't appropriate to test.